source/_patterns/03-organisms/navbar/_navbar.twig

```diff
@@ -0,0 +1,39 @@
+{#
+  Navbar!
+  navbar_classes: String: The html classes for the navbar.
+  navbar_brand_link: String: The URL the brand text will link to.
+  navbar_brand_text: String: The text of the brand.
+  navbar_id: String: The html id for the navbar.
+#}
+
+<nav class="navbar {{ navbar_classes }}">
+  {# Brand #}
+  <a class="navbar-brand" href="{{ navbar_brand_link }}">{{ navbar_brand_text }}</a>
+  {# collapse button #}
+  {% include '@atoms/button/_button.twig' with {
```

Comment:
At the moment this button behaves strangely because of an inconsistency in the Bootstrap design pattern (see image below). I've opened up an issue with bootstrap about it [here](https://github.com/twbs/bootstrap/issues/24968).
![screen shot 2017-12-07 at 1 17 17 pm](https://user-images.githubusercontent.com/19877994/33739590-7f44f764-db52-11e7-8772-1135c974611d.png)